csp/baselib.py

```diff
@@ -212,6 +231,9 @@ def _print_ts(tag: str, x: ts["T"]):
         builtins.print("%s %s:%s" % (t, tag, x))
 
 
+_python_print = print
```

Comment:
I think you left this in, it isn't used anywhere (plus I would not add this to baselib, you can always print in a graph using sys.stdout.write)

Reply:
Also remove the spaces here

Reply:
This is useful if you want to quickly print debug inside baselib, it is not exported in `__all__`

Reply:
If you want to leave this in as a debugging tool, at least add a comment explaining what it's for.